### ISSUE TITLE
no-member with Qt signals using PySide6

### ISSUE THREAD

**Issue by EricEzaM** [PyCQA/pylint#5378]
### Bug description

Exactly the same as #2585.

### Configuration

Create signal_test.py containing:
```python
from PySide6 import QtWidgets
app = QtWidgets.QApplication()
app.focusChanged.connect(lambda x: None)
```

### Command used

```shell
pylint .\signal_test.py --extension-pkg-whitelist=PySide6
```

### Pylint output
```shell
test.py:4:0: E1101: Method 'focusChanged' has no 'connect' member (no-member)
```

### Expected behavior

No Errors

### Pylint version

```shell
pylint 2.11.1
astroid 2.8.4
Python 3.8.10
```

### OS / Environment

Win 10

As mentioned, this was addressed in #2585 for PySide2

### Additional dependencies

PySide6                           6.2.0

**Comment by BoniLindsley** [comment]
I think the check given in <https://github.com/PyCQA/astroid/pull/900#issuecomment-804492415> would fix this. Specifically line 4 of the code snippet.
```py
is_pyside_node = node.qname().partition(".")[0] in ("PySide2", "PySide6")
```
That pull request might need to be adopted with this in mind, or needs to be merged first, for this to be added, I think.

**Comment by adam-grant-hendry** [comment]
I am encountering the same issue with `QAbstractAnimation.finished` signals.

**Comment by adam-grant-hendry** [comment]
> I think the check given in [PyCQA/astroid#900 (comment)](https://github.com/PyCQA/astroid/pull/900#issuecomment-804492415) would fix this. Specifically line 4 of the code snippet.
> 
> ```python
> is_pyside_node = node.qname().partition(".")[0] in ("PySide2", "PySide6")
> ```
> 
> That pull request might need to be adopted with this in mind, or needs to be merged first, for this to be added, I think.

I can confirm that check fixed the problem on my end.

+1 to have the fix merged in PR #900.

**Comment by jacobtylerwalls** [comment]
Duplicate of #4040

(closing now as it's difficult to link two pylint issues to an astroid PR for auto-closing later)
